lua/wikis/marvelrivals/Infobox/Person/Player/Custom.lua

```diff
@@ -0,0 +1,188 @@
+---
+-- @Liquipedia
+-- wiki=marvelrivals
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local CharacterIcon = require('Module:CharacterIcon')
+local CharacterNames = mw.loadData('Module:CharacterNames')
+local GameAppearances = require('Module:GetGameAppearances')
+local Lua = require('Module:Lua')
+local MatchTicker = require('Module:MatchTicker/Custom')
+local Page = require('Module:Page')
+local String = require('Module:StringUtils')
+local Team = require('Module:Team')
+local Variables = require('Module:Variables')
+local Template = require('Module:Template')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local Player = Lua.import('Module:Infobox/Person')
+
+local Widgets = require('Module:Widget/All')
+local Cell = Widgets.Cell
+local Center = Widgets.Center
+
+local ROLES = {

```

Comment:
no keys with spaces or leading numbers so could sugar coat to drop the `['......']` for the keys